src/Components/VideoListingCard/VideoListingCard.css

```diff
@@ -1,10 +1,17 @@
+.card-display-grid {
+  padding: 3rem;
+  display: grid;
+  gap: 2rem;
+  grid-template-columns: repeat(auto-fit, minmax(21rem, 1fr));
```

Comment:
i think `auto-fill`will work better here as when there are less products in a row you don't want them to be spreaded across the availaible space.

Reply:
okay, I will try it out.